### PR TITLE
CN-4879 CN-4880 HodlDeactivateLanding and HodlDeactivationCode screens

### PR DESCRIPTION
**Pull request by ShamanShaulic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/HodlDeactivationCode/HodlDeactivationCode.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <CelText margin={"20 0 0 0"} type={"H4"} align={"center"}>
+          Forgot your code?
+        </CelText>
+        <CelText type={"H4"} align={"center"}>
+          <CelText
+            color={STYLES.COLORS.CELSIUS_BLUE}
+            onPress={() => Linking.openURL("mailto:app@celsius.network")}
+          >
+            Contact our support
+          </CelText>{" "}
```
Comment (fjovakaric) [inline app/components/screens/HodlDeactivationCode/HodlDeactivationCode.js]: this should probably go to Support screen

File: app/components/screens/HodlDeactivateLanding/HodlDeactivateLanding.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+// import HodlDeactivateLandingStyle from "./HodlDeactivateLanding.styles";
+import CelText from "../../atoms/CelText/CelText";
+import RegularLayout from "../../layouts/RegularLayout/RegularLayout";
+import CelButton from "../../atoms/CelButton/CelButton";
+
+@connect(
+  () => ({}),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
+class HodlDeactivateLanding extends Component {
```
Comment (fjovakaric) [inline app/components/screens/HodlDeactivateLanding/HodlDeactivateLanding.js]: should be in HODLLanding screen, with an `if`. One state to activate, another to deactivate

File: app/components/screens/SecuritySettings/SecuritySettings.js [hunk 3]
```diff
@@ -87,13 +91,39 @@ class SecuritySettings extends Component {
...
+
+    if (!isInHodlMode) {
+      actions.navigateTo("VerifyProfile", {
+        onSuccess: () => actions.navigateTo("HodlDeactivateLanding"),
+      });
+    } else {
+      actions.navigateTo("VerifyProfile", {
+        onSuccess: () => actions.navigateTo("HodlLanding"),
+      });
+    }
```
Comment (fjovakaric) [inline app/components/screens/SecuritySettings/SecuritySettings.js]: should go to HODLLanding always, HODLLanding should know if the user is in HODL mode or not
